### ISSUE TITLE
Why where InputBox and HyperlinkBox removed?

### ISSUE THREAD

**Issue by Fruchtzwerg94** [AvaloniaCommunity/MessageBox.Avalonia#144]
After upgrading to the latest prerelease I've noticed InputBox and HyperlinkBox where removed.
Any reason for this? Would be great to keep them in.

**Comment by CreateLab** [comment]
Input box looks bad. Need a lot of time to support normally view of it.
Hyperlink now has no support for mobile platforms. Need special implementation. If u wanna support I'll be grateful

**Comment by CreateLab** [comment]
As an idea - get ability for user to set a callback for hypertext click
i will think about it

**Comment by Fruchtzwerg94** [comment]
I agree that Hyperlink support may is more complicated.
But I am not really happy about the InputBox removed only because of it looks bad. It is a basic control which did his job well in the past. No one is forced to use it like this and users could add their custom implementation as well.
But as a user of the library I love it to get some simple and working boxes without big implementation overhead I need to do.
Please think about adding the InputBox (like it was) again. If it helps, I could also create the PR adding the InputBox again.

**Comment by CreateLab** [comment]
Hyper link returned as param in custom box

**Comment by anovik** [comment]
It would be great to have input control back if this is possible

**Comment by gusmanb** [comment]
I have created a pull request that restores the input capability and adds the hyperlink capability to the standard messagebox. Instead of creating a new type of messagebox the functionality is optional for both messagebox types. It is activated if InputParameters are provided in the xxxMessageBoxParameters class, like hyperlink.

Also, I have added a viewmodel static factory to enable proper design mode views.
